core/models.py

```diff
@@ -862,6 +862,16 @@ class Place(models.Model):
     latitude = models.FloatField(null=True)
     longitude = models.FloatField(null=True)
 
+    @property
+    def get_city_county(self):
+        if self.city and self.county:
+            return "{}, {}".format(self.city, self.county)
+        if self.city:
+            return self.city
+        if self.county:
+            return self.county
+        return "No city or county found"
```

Comment:
If this text is going to appear in fairly narrow tables, it may be better to have something short like "UNKNOWN" or "N/A"